ui/src/app/pages/workspace/workspace-library.tsx

```diff
@@ -97,9 +103,10 @@ interface ReloadableProfile {
 }
 
 interface CurrentTab {
-  title: string;
-  icon: string;
+  description?: string | JSX.Element;
```

Comment:
can we just make this `JSX.Element`?

Reply:
I think I am fine with that, but I want the option to use a string to be there. We can leave it out for now and add that eventually.